plugins/PermissionsEx/permissions.yml

```diff
@@ -113,6 +113,7 @@ groups:
     - permissions.user.promote.default
     - permissions.user.rank.demote
     - permissions.user.rank.promote
+    - logblock.lookup
```

Comment:
Add `logblock.tools.tool` and `logblock.tools.toolblock`

From the wiki:
- logblock.lookup
> Grants user access to /lb lookup (Users can't use their tools without this node!)